README.md

```diff
@@ -2,54 +2,79 @@
 
 # NGINX Modern Reference Architectures
 
-This repository has the basics for a common way to deploy and manage a modern app. Over time we'll be building more 
-example architectures using different deployment models and options including other clouds, and you’ll be able to find
-those from here.
+A repository for modern application deployment and management, 
+created to illustrate modern app design principles and encourage 
+our community to deploy on Kubernetes. This project provides 
```

Comment:
Is there a way to avoid phrasing that hints that the community is possessed by us?

Reply:
I believe so – in what way does it feel possessed?

Reply:
Hmm I can see that (though I'm also weird) – easy fix would be "encourage the [insert name] community"